README.md

```diff
@@ -240,6 +241,10 @@ Use `0` instead of `none` to specify that a style has no border.
     }
     ```
 
+### Variables
+
+Prefer dash-cased variable names (e.g. `$my-variable`) over camelCased or snake_cased variable names. It is acceptable to prefix variable names that are intended to be used only within the same file with an underscore (e.g. `$_my-variable`).
```

Comment:
If I want to go remove a _-prefixed variable, I can't simply assume that it's not being used - I still have to grep for it just the same as if it was intended to be public. Therefore, the _ isn't adding anything more than a comment would.